Lib/test/test_copy_fallback.py

```diff
@@ -0,0 +1,60 @@
+"""Unit tests for the copy module without the C accelerator."""
+
+from test.support.import_helper import import_fresh_module
+
+copy_py = import_fresh_module('copy', blocked=['_copy'])
+
+import unittest
+import sys
+
+from test.support.import_helper import import_fresh_module
+
+
+TESTS = 'test.test_copy'
+
+def load_tests(loader, tests, pattern):
+    try:
+        pure_tests = import_fresh_module(TESTS, fresh=['copy'],
+                                        blocked=['_copy'])
+    finally:
+        # XXX: import_fresh_module() is supposed to leave sys.module cache untouched,
+        # XXX: but it does not, so we have to cleanup ourselves.
```

Comment:
Please open a new bug report for this issue.

Reply:
Not sure this is valid, it was copied from a patch from 2016

Reply:
This was from the original patch (2016), not sure this still applied.